hw/ip/spi_host/data/spi_host.hjson

```diff
@@ -0,0 +1,590 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{ name: "pwm",
+  clock_primary: "clk_i",
+  other_clock_list: [ "clk_core_i" ],
+  reset_primary: "rst_ni",
+  other_reset_list: [ "rst_core_ni" ],
+  bus_device: "tlul",
+  bus_host: "none",
+  regwidth: "32",
+  param_list: [
+    { name: "ByteOrder",
+      desc: '''Byte order to use when transmitting or receiving data.  If ByteOrder = 0,
+               the IP uses a Big-Endian ordering for the bytes in TXDATA and RXDATA.
+               The most significant byte (MSB) of TXDATA is transmitted first, and
+               received data is placed in the MSB location of RXDATA.  If ByteOrder = 1,
+               a Little-Endian ordering is used for these registers, and the LSB of each
+               gets priority for receiving and transmitting data.'''
+      type: "int",
+      default: "0"
+    },
+    { name: "MaxCS",
+      desc: "The number of active-low chip select (cs_n) lines to create. (Maximum 8)",
+      type: "int",
+      default: "8"
+    },
+  ],
+  available_output_list: [
```

Comment:
this is a bit nit picky, but i dont think you need to declare the `_o` suffix, that's automatically handled i think.

Reply:
I think you are right.  (I wish it didn't though.  That makes things a bit messy, and inconsistent here and there.)

Reply:
One thing that is less nit-picky: the sd lines should all be labelled as "inout"'s.  Changed.